app/controllers/api/saved_jobs_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Api::SavedJobsController < ApplicationController
+  before_action :authorize!, only: [:index, :show]
+
+  def index
+    jobs_to_look_up = current_user.saved_jobs
+    jobs_to_send = look_up(jobs_to_look_up)
+    render json: jobs_to_send
+  end
+
+  def show
+    job = IndeedApi.search_for_job(params[:job_key])
+    if job["results"]&.length == 1
```

Comment:
it shouldn't return more than one I just wanted to be safe just in case something weird happened on Indeed's side. I will change that to an empty check.